native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar-test.ios.js

```diff
@@ -123,5 +123,45 @@ describe('ios', () => {
         .toJSON();
       expect(tree).toMatchSnapshot();
     });
+
+    it('should render correctly with a leading icon in the title', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBar
+            title={{ value: 'Checkout', icon: 'lock', iconPosition: 'leading' }}
+            leadingButton={
+              <BpkNavigationBarBackButtonIOS
+                title="Back"
+                showTitle
+                onPress={jest.fn()}
+              />
+            }
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with a trailing icon in the title', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBar
+            title={{
+              value: 'Checkout',
+              icon: 'lock',
+              iconPosition: 'trailing',
+            }}
+            leadingButton={
+              <BpkNavigationBarBackButtonIOS
+                title="Back"
+                showTitle
+                onPress={jest.fn()}
+              />
+            }
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
```

Comment:
Any reason why these tests are not in common? The additional trailing icon test is now missing from android, for instance.